docs/guide/static-deploy.md

```diff
@@ -33,11 +33,11 @@ By default, the build output will be placed at `dist`. You may deploy this `dist
 
 ### Testing The App Locally
 
-Once you've built the app, you may test it locally by running `npm run preview` command.
+Once you've built the app, you may test it locally by running `npm run serve` command.
 
 ```bash
 $ npm run build
-$ npm run preview
+$ npm run serve 
 ```
 
 The `preview` command will boot up local static web server that serves the files from `dist` at http://localhost:5000. It's an easy way to check if the production build looks OK in your local environment.
```

Comment:
"The `preview` command will boot up local static web server"

Do you think this should say `serve` or leave as is in context to `vite preview`?

Reply:
We could change it to
```
The `vite preview` command...
```
so it is less confusing